_layouts/posts.html

```diff
@@ -15,6 +15,13 @@
 <body>
 <main>
   <p class="no-print"><a href="{{ site.baseurl }}/">< Back to Home</a></p>
+  {% if page.tags %}
+  <div class="tags">
```

Comment:
I'd say no because it's not a child of the sibling paragraph above it. If it is rendered, it will look nested beneath the `<p>`.